src/components/cms/BrowseSkill/Carousal.css

```diff
@@ -0,0 +1,21 @@
+.slick-slide {
```

Comment:
We already have slick styles used in ChatApp.css. KIndly remove both and move it to a higher scope level.